code/datums/ASRS.dm

```diff
@@ -138,13 +112,23 @@
 	group = "ASRS"
 
 //===================================
-//speciality armor
-/datum/supply_packs/armor_leader/asrs
+// Materials
+/datum/supply_packs/sandbags/asrs
 	buyable = 0
 	group = "ASRS"
-	cost = ASRS_LOWEST_WEIGHT
+	cost = ASRS_VERY_LOW_WEIGHT
 
-/datum/supply_packs/armor_rto/asrs
+/datum/supply_packs/sandbagskit/asrs
 	buyable = 0
 	group = "ASRS"
-	cost = ASRS_LOWEST_WEIGHT
+	cost = ASRS_VERY_LOW_WEIGHT
+
+/datum/supply_packs/metal/asrs
+	buyable = 0
+	group = "ASRS"
+	cost = ASRS_VERY_LOW_WEIGHT
+
+/datum/supply_packs/plas/asrs
+	buyable = 0
+	group = "ASRS"
```

Comment:
materials crates are undesirable in free ASRS as they encourage big fob spam
current metal/material economy is fine, no need to add in free material crates
would advise replacing these with maybe flashlights/flares/fire extinguishers/something else if you want